integration/lifecycle/capabilities_endorse_test.go

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
Is there any specific handling for ed25519 that influences lifecycle specifically?

Reply:
No. I just put it in the lifecycle folder because it seemed the most related integration test "family", considering your previous  #suggestions:
> I think a **single** integration test that:
> 
> * Has peers and orderers in it
> * Transacts on a channel
> * The channel contains both ECDSA and ed25519 identities and they're both used in the endorsement policy
> 
> Is sufficient.

Reply:
i see, ok